src/client/components/dragAndDrop/drag-and-drop.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import Dropzone from 'react-dropzone';
+import { createAudioData, playPauseSoundFromFile } from '../../actions/audioActions';
+
+import './drag-and-drop.css';
+
+
+class DragAndDrop extends Component {
+    onDrop = (acceptedFiles) => {
+      const file = acceptedFiles[0];
+      const audioFile = new Audio(URL.createObjectURL(file));
+      const sound = new Pizzicato.Sound({
+        source: 'file',
+        options: {
+          path: audioFile.src,
+          loop: true,
+        },
+      }, () => this.createSoundInfoInState(sound, file));
+    };
+
+  createSoundInfoInState = (sound, file) => {
+    const { audioData: { analyser }, createAudioDataAsProp } = this.props;
+    sound.connect(analyser);
+    createAudioDataAsProp({
+      sound,
+      trackName: file.name,
+      trackType: file.type,
+      trackSize: file.size,
+    });
+  };
+
+  render() {
+    const maxSize = 1000000000;
+    return (
+      <div className="container text-center mt-5">
```

Comment:
I think it's not a good idea add a name like text-center and mt-5

Reply:
@StanislavFedyna MARGIN-TOP: 5px; you understand me?

Reply:
change this 💩
We don't use a style like in bootstrap or something like that
If you need margin, you can use margin-top or text-align: center for centering of text

Reply:
done